live-examples/js-examples/proxyhandler-has.html

```diff
@@ -0,0 +1,32 @@
+<pre>
+<code id="static-js">var handler1 = {
+  has (target, key) {
+    if (key[0] === '_') {
+      return false;
+    }
+    return key in target;
+  }
+}
+
+function invariant (key, action) {
+  if (key[0] === '_') {
+    throw new Error(`Invalid attempt to ${action} private "${key}" property`);
+  }
+}
```

Comment:
I think you don't need the `invariant` function?

Reply:
Good call :) I have some minor continuity changes to push from this morning - the above issue should be resolved in that.

Reply:
oops - I removed the reference but not the invariant. Blame the monsters.